src/status_providers/github.ts

```diff
@@ -81,9 +88,18 @@ export class GithubStatusProvider extends BaseStatusProvider {
       ) {
         // Edge case, this is what GitHub returns when there are no registered legacy checks.
         logger.debug('No legacy checks detected, checking for runs...');
+        const hasPendingActiveSuites = revisionCheckSuites.check_suites.some(
+          suite =>
+            // Need the any cast as octokit lacks this prop in its types
+            (suite as any).latest_check_runs_count > 0 &&
+            suite.status === 'queued'
```

Comment:
_nit_: This is done in two places and might qualify for a small helper function given the comment.